packages/opencensus-exporter-stackdriver/src/stackdriver-monitoring-utils.ts

```diff
@@ -216,6 +228,32 @@ function createBucketCounts(buckets: DistributionBucket[]): number[] {
   return bucketCounts;
 }
 
+/** Converts a OpenCensus Buckets to a list of proto Exemplars. */
+function createExemplars(buckets: DistributionBucket[]): Exemplar[] {
+  return buckets.filter((bucket => !!bucket.exemplar))
+      .map((bucket) => ({
+             value: bucket.exemplar!.value,
+             timestamp: toISOString(bucket.exemplar!.timestamp),
+             attachments: Object.keys(bucket.exemplar!.attachments)
+                              .map((key) => {
```

Comment:
What would you think about breaking out a helper function to map from a bucket attachment key to an exemplar?

Reply:
Sure, Done.